### ISSUE TITLE
Can someone tell me the status of backing up and restoring to external media?

### ISSUE THREAD

**Issue by 2011** [jensstein/oandbackup#270]
I have seen a few issues (from some time back) in the past regarding this, but nothing indicating a resolution (and I still get error messages in the app when attempting to desginate the directory for storing backups on external media. The ability to back up to external media seems so basic in an Android backup app that it completely baffles me as to why oandbackup doesn't support it. The techniques to enable that seem relatively straightforward for programmers (a category into which, unfortunately, I do not fit). I consider the capability of backing up to external media an absolute requirement (and the inability to do so a complete dealbreaker).

**Comment by marcdw1289** [comment]
On three of my devices I multiboot ROMs. I had Titanium Backup use a certain directory for backups on the SD card. Had to use the document provider to access the card, otherwise I couldn't write. I also had TiBu's setting in a shared location. Problem with that is that each ROM would identify the SD card differently and I had to keep setting the backup location.

Awhile back I read the TiBu docs and recently some notes for SyncThing. The solution is to use a directory on the SD card that is owned by the app. Today I tried a test with oandbackup and backed up fine to external SD card.

If not already exists, on the SD card create /Android/data/dk.jens.backup/ directory. For me it already existed with a 'files' subdirectory in it. Not knowing what that subdirectory is for I created an 'oandbackups' subdir. Then in oandbackup I selected the following path for backups...

/storage/5D77-83B1/Android/data/dk.jens.backup/oandbackups

Did a couple of sample backups and it worked just fine. One can also use /Android/media/ instead. That also works and might be preferable since it's less "crowded".

In short, copy your existing oandbackups directory to one of the app-owned paths and test. Should work.

**Comment by nicoursi** [comment]
I can conferma this method works 😊
I wish there was more development on this project which looks like slowly dying
